scenario.py

```diff
@@ -60,11 +60,19 @@ def __init__(self, params, experiment_path):
 
         self.federated_test_score = int
 
+        # Define how federated learning aggregation steps are weighted. Toggle between 'uniform' and 'data-volume'
+        # Default is 'uniform'
+        if "aggregation_weighting" in params:
+            self.aggregation_weighting = params["aggregation_weighting"]
+        else:
+            self.aggregation_weighting = "uniform"
+
         self.node_list = []
 
         self.contributivity_list = []
 
         self.epoch_count = 40
+        self.minibatch_count = 20
```

Comment:
I was wondering if we should use ratio or absolute values but I think absolute values is better.
Also we probably need an easy way to specify to not use mini batch at all.

Reply:
I thought about that: I am not sure. Actually, our distributed learning algo doesn't really make sense without the iteration over mini-batches. Iterating only over epochs is really too simplistic, and I don't think we want a parameter to enable the user to do it.